src/main/java/uk/gov/companieshouse/acsp/manage/users/controller/UserAcspMembership.java

```diff
@@ -37,11 +50,20 @@ public ResponseEntity<AcspMembership> getAcspMembershipForAcspId(
 
     @Override
     public ResponseEntity<List<AcspMembership>> getAcspMembershipForUserId(
-            @NotNull String xRequestId,
-            @NotNull String ericIdentity,
-            @Valid Boolean includeRemoved
+            final String xRequestId,
+            final String ericIdentity,
+            final Boolean includeRemoved
     ) {
-        return null; // TODO(https://companieshouse.atlassian.net/browse/IDVA6-1145)
+        final boolean excludeRemoved = includeRemoved == null || !includeRemoved;
+
+        final List<AcspMembership> memberships = acspMembershipListMapper.daoToDto(
```

Comment:
mappers should be inside service.
service should call the mappers and provide the data back to the controller.
"segregation of concern"